plugins/CoreVisualizations/templates/_dataTableViz_htmlTable_ratio.twig

```diff
@@ -1,20 +1,19 @@
 {% if column in properties.report_ratio_columns and (column in totals|keys or forceZero|default) -%}
     {% set reportTotal   = totals[column]|default(0) %}
-    {% if siteTotalRow|default is not empty %}
+
+    {% if siteTotalRow is defined and siteTotalRow is not empty %}
```

Comment:
@diosmosis  do you understand why we calculate the percentage differently 
* The percentage value we actually show is based on reportTotal
* The percentage value in tooltip is based on siteTotal

I wonder if we can't always use the same logic? Eg always use reportTotal? Then we wouldn't even need to do the API call in the HTML table maybe. I suppose the reportTotal may be formatted as well though. Actually I think I get it... the siteTotal is without segment while reportTotal may have a segment...

Looks good to merge @diosmosis

Reply:
> Actually I think I get it... the siteTotal is without segment while reportTotal may have a segment...

Interesting, I had no idea why this was there :)

Reply:
Actually, I just tested it and it does actually not seem to be the case. I reckon the `siteSummary` might not be needed at all and we can just use the same percentage as we show in the table based on reportTotal?

Reply:
Looks like this is the commit that differentiated between totals & siteSummary: https://github.com/matomo-org/matomo/commit/bb0e02d21541125eca23ba0256b3a3cde3db262c

Maybe `siteSummary` was supposed to get the data w/o segment, but because we weren't using `Request::processRequest(..., $default = [])` or something similar, it wasn't working as expected? I wonder if this is actually a bug and we can fix it by making siteSummary work the way you thought it worked.

Reply:
Sounds great to add the default 👍